MVVM Module/Source/Feed Presentation/FeedViewModel.swift

```diff
@@ -20,14 +20,23 @@ final class FeedViewModel {
 
 	var onLoadingStateChange: Observer<Bool>?
 	var onFeedLoad: Observer<[FeedImage]>?
+	var onErrorMessage: Observer<String?>?
```

Comment:
Suggestion: 
`onErrorMessage` can be called `onErrorStateChange` to be consistent with `onLoadingStateChange`
```suggestion
	var onErrorStateChange: Observer<String?>?
```

Reply:
Thanks @danillahtin . I've pushed the changes.